tests/regression/test_disk_full.py

```diff
@@ -0,0 +1,166 @@
+import pytest
+import queue
+import logging
+from unittest.mock import patch
+from pay.payment_producer import PaymentProducer
+from pay.payment_consumer import PaymentConsumer
+from tests.utils import Args, make_config
+from plugins import plugins
+from Constants import RunMode
+from cli.client_manager import ClientManager
+from NetworkConfiguration import init_network_config
+from model.baking_dirs import BakingDirs
+from model.baking_conf import BakingConf
+from api.provider_factory import ProviderFactory
+from config.yaml_baking_conf_parser import BakingYamlConfParser
+from calc.service_fee_calculator import ServiceFeeCalculator
+from util.process_life_cycle import ProcessLifeCycle
+
+logger = logging.getLogger("unittesting")
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+baking_config = make_config(
+    "tz1gtHbmBF3TSebsgJfJPvUB2e9x8EDeNm6V",
+    "tz1gtHbmBF3TSebsgJfJPvUB2e9x8EDeNm6V",
+    10,
+    0,
+)
+
+
+@pytest.fixture
+def args():
+    # Test with PRPC node
+    args = Args(
+        initial_cycle=10,
+        reward_data_provider="tzkt",
+        api_base_url="https://api.tzkt.io/v1/",
```

Comment:
I don't want any URLs in the tests. They should all be taken from constants.py! We are also again mixing testnet and mainnet. I would vote to do the tests only on mainner and not on testnet if there are especially tests that invole a specific block level etc